messages/store.json

```
@@ -141,13 +141,29 @@
             "regExtensionNameFlagDescription": "Unique name for extension",
             "savingConfigIntoConfig": "Successfully mapped the extension",
             "storeId": "Optional webstore ID flag",
+            "errorEPN": "Invalid EPN. Run [commerce:extension:points:list] for all valid EPN values",
             "nonexistent": "Can't find extension with registered extension name:",
+            "previousEPN": "Unmapped the extension:",
```

super nit but should we remove the colon since the error message has more info afterwards?
Right now it looks like: `Unmapped the extension: 'JinsuTicket' because it was for the same EPN in this webstore`
```suggestion
            "previousEPN": "Unmapped the extension",
```